Code & Coffee Cape Town | Home and About pages

aamir-asaram opened the pull request:
# Highlights
**Home Page - Mobile**

- [x] Responsive hamburger button
- [x] Hover animation on menu items 
- [x] Menu items link to different sections on the home and about pages
- [x] Background image behind the title for aesthetic purposes
- [x] Custom background texture in **main program** section
- [x] Hover animation on the join button
- [x] Dynamically generated cards for the speakers section
- [x] Custom background images behind speaker images [coffee bean for coffee speakers and binary for code speakers]. 
- [x] Responsive more/less button to show/hide additional speakers
- [x] Responsive logo icons with hover animation and links to sponsor pages  

**Home Page - Desktop**

- [x] Responsive resizing of the mobile version
- [x] Header with hover effect on items
- [x] Navbar with links to different sections on home and about pages

**About Page - Desktop**

- [x] Logo in navbar links to home page
- [x] Article in the logo section contains a link to the logo designer.
- [x] Sponsors section is hidden and the color theme of the footer changes when switching from mobile to desktop 

**General**

- [x] Added tab icon [works better with light theme].

## Future Features

- [ ] Tickets Page
- [ ] Schedule Page

aamir-asaram committed:
Add intro section and navbar

aamir-asaram committed:
Add main program section

aamir-asaram committed:
Add featured speakers section

aamir-asaram committed:
Add about.html

aamir-asaram committed:
broke horizontal scroll

aamir-asaram committed:
Remove sponsors section

aamir-asaram committed:
Remove sponsors styling

aamir-asaram committed:
Restart before sponsors

aamir-asaram committed:
Finish mobile about page

aamir-asaram committed:
Create CNAME

aamir-asaram committed:
Delete CNAME

aamir-asaram committed:
Update background images

aamir-asaram committed:
Merge branch 'mobile-home' of github.com:aamir-asaram/capstone-one into mobile-home

aamir-asaram committed:
Add some navbar function

aamir-asaram committed:
About page navbar

aamir-asaram committed:
Adapt home page to desktop

aamir-asaram committed:
Add desktop layout for about page

aamir-asaram committed:
Add dynamic generation to speakers section

aamir-asaram committed:
Add tab icon

aamir-asaram committed:
Implement more button functionality

aamir-asaram committed:
Add logo to README

aamir-asaram committed:
Add arrow icon to more/less button

aamir-asaram committed:
Add sponsor section to mobile about page

aamir-asaram committed:
Add link to about page logo

aamir-asaram committed:
Add speakers and logo

aamir-asaram committed:
Add linters and update README

aamir-asaram committed:
Add linter

aamir-asaram committed:
Fix eslint errors

aamir-asaram committed:
Fix some style lint errors

aamir-asaram committed:
Fix outlines moving surrounding content in program section

aamir-asaram committed:
Fix style and eslint errors

aamir-asaram committed:
Fix style errors

wayungi commented on README.md:
```diff
@@ -53,9 +49,8 @@ This is a showcase webpage for an event called Code & Coffee. It closely mirrors
 
 > Describe between 1-3 key features of the application.
```
- [ ] Please remove all these guiding texts from your readme
![inv](https://github.com/aamir-asaram/capstone-one/assets/99656687/4e5b67be-4667-49db-9cd8-706a078f8c93)

wayungi reviewed (commented):
Hi @aamir-asaram , 
![image](https://user-images.githubusercontent.com/99656687/224543512-ddbefa59-dc68-47c2-b7fe-f55acdacd3eb.png)

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project with an unprofessional README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

aamir-asaram committed:
Remove unnecessary comments from README file

aamir-asaram committed:
Update README.md

iambenkis commented on README.md:
```diff
@@ -162,11 +150,7 @@ If you like this project please leave a star!
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
+I would like to thank my sister <a href="https://www.instagram.com/aaliadrawsthings/">@aaliadrawsthings</a> for designing the logo and custom backgrounds.
```
- [x] As you have used someone else design, it is important to give a credit to him/her as a privilege. This helps anyone who visits your website but is interested in the design you have followed get to know the source 💡. Moreover, it is among the project requirements to give credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design you used. Please add her here in the Acknowledgement section and the link to the design. 😃 👍

iambenkis commented on main.js:
```diff
@@ -0,0 +1,139 @@
+    container.classList.add('speaker-content');
+    const speakerCard = document.createElement('div');
+    speakerCard.classList.add('speaker');
+    speakerCard.innerHTML = `
+      <img src="${speakers[i].img}" alt="${speakers[i].name}">
+      <section>
+        <h3>${speakers[i].name}</h3>
+        <p class="subheading">${speakers[i].title}</p>
+        <hr>
+        <article>${speakers[i].bio}</article>
+      </section>
```
- [x] Good implementation of the speaker section using JavaScript to perform this action 👏 , There is some point you still need to improve on the design. You will notice with me that the part in rounded in red on picture below should stick to the Figma design. You use two kind in your web page. Kindly use the same thing as in the given design 👍

| **Your design** |  **Cindy Design** |
| :---: | :---: |
| ![Capture1](https://github.com/aamir-asaram/capstone-one/assets/80458199/3666a9cc-4b07-40c4-86e3-160c3025a383) | ![Capture](https://github.com/aamir-asaram/capstone-one/assets/80458199/8736c8fa-969a-400b-863a-56beb33d69b0) |

aamir-asaram replied:
Hi @iambenkis, I appreciate the review! The use of two kinds of background was intentional as the coffee beans represent a coffee-related speaker and the binary represents a code-related speaker [I mentioned this in the PR description]. As per the design guideline I was under the impression we could change pictures slightly so is it okay if I keep this change? Or is it necessary that I change it back? Thanks in advance :smile:

iambenkis commented on index.html:
```diff
@@ -0,0 +1,118 @@
+      </div>
+    </nav>
+
+    <section class="intro">
+      <p>"Hello! Programmers"</p>
+      <h1>Code & Coffee <br>Cape Town</br> 2023</h1>
```
Kindly make sure the font you use in your design look exactly the same as in the provided design 😄 , 
Even though the text should be customized it should look the same , it could be better if you improve that 👍 

| Your design | Cindy Design |
| :---: | :---| 
| ![Capture1](https://github.com/aamir-asaram/capstone-one/assets/80458199/7c1de9e3-80bd-4b9e-a274-ba08cb527d2a) |  ![Capture](https://github.com/aamir-asaram/capstone-one/assets/80458199/96329faf-40dc-4a08-93b2-c93611364c6c) |


The same comment is applied to the `about` page , which I think also should be customized 👍 

| Your design | Cindy Design |
| :---: | :---| 
| ![Capture1](https://github.com/aamir-asaram/capstone-one/assets/80458199/867e8225-5c01-455a-8ea8-6e4b5b59e572) | ![Capture](https://github.com/aamir-asaram/capstone-one/assets/80458199/4f7fc2dc-abb8-4f78-a369-5d86045303f8) |

aamir-asaram replied:
![font](https://github.com/aamir-asaram/capstone-one/assets/76215321/88f2f757-b262-4641-b83f-984e7e86c064)
Also @iambenkis I've found a font that seems to match the template closely but it doesn't seem to have the & symbol or any numbers - kindly advise if it is okay for me to remove the year from the title text.

iambenkis reviewed (changes requested):
Hello @aamir-asaram  , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃 

## Highlights 💪 
- Innovative idea in a good-looking website
- good UI design
- no linter errors
- correct GitHub flow is used
- good PR description


## Required Changes ♻️

_Check other comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aamir-asaram committed:
Change font-family of h1 to Cocogoose [per design template], update readme to include acknowledgement and changed speaker background as per CR

aamir-asaram committed:
Merge branch 'mobile-home' of github.com:aamir-asaram/capstone-one into mobile-home

aamir-asaram commented:
### Changes Made
- [x] Changed title font to Cocogoose to more closely match the original design
- [x] Added acknowledgment of the original creator to the appropriate section in the README

Although I do not agree with the previous code reviewer, I have changed the backgrounds of the speaker section to more closely match the original design. I feel that my original design did not differ too much from the original while still adding some personalization. Kindly advise if my previous design was too different from the original to be considered a "small adjustment to the text or images" as permitted by the guideline for the project or if it's okay for me to revert to my original backgrounds.

Munsa1 reviewed (approved):
Hi @aamir-asaram, 
![congratulations-2](https://github.com/aamir-asaram/capstone-one/assets/81932478/87b40865-df27-499c-bc91-3827e79eb20f)

## Highlights
Nice Speaker Section Button Functionality(`More` and `Less`) :heavy_check_mark: 
Nice Click Effect For Past Events Section :heavy_check_mark: 
Made Requested Changes :heavy_check_mark: 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
